cdap-ui/app/features/hydrator/services/create/stores/config-store.js

```diff
@@ -234,6 +232,14 @@ class ConfigStore {
     }
     this.emitChange();
   }
+  setEngine(engine) {
+    if (this.state.config.artifact === this.GLOBALS.etlBatch) {
+      this.state.config.engine = engine || 'mapreduce';
+    }
+  }
+  getEngine() {
+    return this.state.config.engine || 'mapreduce';
```

Comment:
do we need to add a check for etlBatch as well here? Otherwise it can actually return mapreduce for etlRealtime, right?